tests/test_consent/api/test_views.py

```diff
@@ -548,6 +556,46 @@ def test_consent_api_get_endpoint(self, factory, items, request_body, expected_r
         self._assert_expectations(response, expected_response_body, expected_status_code)
 
     @ddt.data(
+        (
+            factories.UserDataSharingConsentAuditFactory,
+            [{
+                'user__user_id': TEST_USER_ID,
+                'user__enterprise_customer__uuid': TEST_UUID,
+                'user__enterprise_customer__enforce_data_sharing_consent': 'at_enrollment',
+                'state': 'disabled'
+            }],
+            {
+                DSCView.REQUIRED_PARAM_USERNAME: TEST_USERNAME,
+                DSCView.REQUIRED_PARAM_ENTERPRISE_CUSTOMER: TEST_UUID,
+                DSCView.REQUIRED_PARAM_COURSE_ID: TEST_COURSE,
+            },
+            {
+                DSCView.REQUIRED_PARAM_USERNAME: TEST_USERNAME,
+                DSCView.REQUIRED_PARAM_ENTERPRISE_CUSTOMER: TEST_UUID,
+                DSCView.REQUIRED_PARAM_COURSE_ID: TEST_COURSE,
+                DSCView.CONSENT_EXISTS: True,
+                DSCView.CONSENT_GRANTED: False,
+                DSCView.CONSENT_REQUIRED: False,
+            },
+            200
+        ),
+    )
+    @ddt.unpack
+    def test_consent_api_get_endpoint_course_not_in_catalog(
+            self,
+            factory,
+            items,
+            request_body,
+            expected_response_body,
+            expected_status_code
+    ):
+        self.cd_client.course_in_catalog.return_value = False
+        if factory:
```

Comment:
@haikuginger The data above doesn't seem to include a case where this would evaluate to `False`?